sipi/scripts/sipi.init.lua

```diff
@@ -20,14 +20,16 @@ require "get_knora_session"
 --    filepath: server-path where the master file is located
 -------------------------------------------------------------------------------
 function pre_flight(prefix, identifier, cookie)
-    server.log("pre_flight called in sipi.init.lua", server.loglevel.LOG_DEBUG)
+    server.log(os.date("!%Y-%m-%dT%H:%M:%S") .. " - pre_flight called in sipi.init.lua", server.loglevel.LOG_DEBUG)
```

Comment:
Would it be possible to also have miliseconds in the timestamp? This would later be useful when it comes to measuring durations (metrics).

Reply:
not without using a different library, which I wanted to avoid

Reply:
this proved to be too annoying because in our Lua setup we don't have a package manager or anything. So I'd rather leave it out for now.  
If you need millisecond precise time measurements, that is however possible:

```lua
before = os.clock()
-- do the work
after = os.clock()
duration = (after - before) * 1000
log("work took " .. duration .. "ms")
```